ros2/plugin_aspects.bzl

```diff
@@ -38,15 +38,16 @@ Ros2PluginCollectorAspectInfo = provider(
     ],
 )
 
-_ROS2_COLLECTOR_ATTR_ASPECTS = ["data", "deps"]
+# From https://github.com/bazelbuild/rules_python/blob/0.33.2/python/config_settings/transition.bzl#L121-L140
```

Comment:
I just updated rules_python to 1.1.0. main branch should be merged in here.

Reply:
done